### PR TITLE
Create Docker documentation

### PR DESCRIPTION
**Pull request by ralfting**

### CODE REVIEW

File: _contents/docker.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Why choose?
+
+Docker's main objective is to build, deliver and run applications.
+Compared with Vagrant, Docker is lighter and more portable. Vagrant is focused on creating environment to develop , while Docker on production environment. Using Vagrant to create our development environment, it become sometimes hard to update itself in sync with production server, however Docker is focused in deploy into production and to environment sharing, facilitating a lot the update between development environment to production environment and sharing with all team.
+
+
+## Documentation
+
+https://docs.docker.com/
```
Comment (wkurosawa) [inline _contents/docker.md]: @ralfting can you make this site address a hyperlink? I'm going to fix this in Rails page.

File: _contents/docker.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: content
+title:  "Docker"
+permalink: /contents/docker
+description: Docker is a platform created to build, deploy and run application in isolated environment.
```
Comment (wkurosawa) [inline _contents/docker.md]: Even thought Docker is described here in this place, I miss a piece of text right before the **Why** part... 
Exclusively when talking about Docker, that is not often used in Flama yet. 
Maybe just copy and detail a little further with a H2.

File: _contents/docker.md (hunk 1 above)
Comment (wkurosawa) [inline _contents/docker.md]: Also.. what do you think of listing along with this institutional documentation other references? 
like tutorials, guides, examples..

Comment (ralfting) [inline _contents/docker.md]: Hey @anapaulavnoronha, I fixed this in the last commit [Remove complexity the text](https://github.com/flama/code-toolkit/commit/7d5ddd1a6c0251a6a2cc3a17db87737919a7f5c3) and  [Add 'What is' in docker documentation](https://github.com/flama/code-toolkit/commit/7a3066f7c546c389401a049587241512622ffeec)

**Pull request merged**